css/estilo.css

```diff
@@ -0,0 +1,92 @@
+body{
+	font-family: 'Open Sans', sans-serif;
+	text-align: center;
+}
+/* grid 960px */
+.linha{
+	width: 960px;
+	margin: 0 auto;
+	overflow: auto;
+	padding: 5px 0px;
+}
+.coluna{
+	padding: 0px 10px;
+	float: left;
+}
+.col1{
+	width: 60px;
+}
+.col2{
+	width: 140px;
+}
+.col3{
+	width: 220px;
+}
+.col4{
+	width: 300px;
+}
+.col5{
+	width: 380px;
+}
+.col6{
+	width: 460px;
+}
+.col7{
+	width: 540px;
+}
+.col8{
+	width: 620px;
+}
+.col9{
+	width: 700px;
+}
+.col10{
+	width: 780px;
+}
+.col11{
+	width: 860px;
+}
+.col12{
+	width: 940px;
+}
+/* Estilos reutilizaveis*/
+header{
+	background: #2C3E50;
+	padding: 45px 0px 45px 0px;
+}
+h1{
+	color: #ebebeb;
+	font-size: 42px;
+}
+a.ui-button{
+	text-decoration: none;
+	font-weight: bold;
+	color: #2C3E50
+}
+a.contained{
```

Comment:
conforme o site for ficando maior, nós teremos varias classes, por isso é importante nós sermos explícitos no nome de cada uma. `.contained` é um nome muito genérico, por isso seria melhor chamá-lo de `.ui-button--contained`.
O `--` se refere a um modifier para o estilo original.
Por exemplo:

```
.ui-input {}
.ui-input--error {}
.ui-input--success {}
```
Dá uma olhada nesse post aqui que explica com mais detalhes:
https://tableless.com.br/bem-um-novo-metodo-para-seu-css/